src/gauge/radial-gauge/radial-gauge.component.html

```diff
@@ -0,0 +1,68 @@
+<ngx-charts-chart
+  [view]="[width, height]">
+
+  <svg:g [attr.transform]="getTranslate()">
+
+      <!-- arcs -->
+
+      <svg:g *ngFor="let arc of arcs" class="arc">
```

Comment:
nit pick, but it looks like the other html templates are a consistent 2 spacing and this has some 4 spacing's intermixed with 2. I would go with the 2 for consistency.